src/sentry/migrations/0099_fix_project_platforms.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-09-02 21:33
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations
+from django.db.models import Q
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def fix_project_platform(apps, schema_editor):
+    """
+    Find projects whose platform is either 'python-tracing' or 'node-tracing',
+    and change them to be either 'python' or 'node' respectively.
+    """
+    Organization = apps.get_model("sentry", "Organization")
+    Project = apps.get_model("sentry", "Project")
+
+    for org in RangeQuerySetWrapperWithProgressBar(Organization.objects.filter(status=0)):
+        # We migrate a project at a time, but we prefer to group by org so that for the
+        # most part an org will see the changes all at once.
+        for project in Project.objects.filter(
+            Q(platform="node-tracing") | Q(platform="python-tracing"), organization=org, status=0
+        ):
```

Comment:
We don't have an index on `platform`. Typically when this is the case we just bring in all rows and iterate over them manually.

Reply:
Would explicitly checking for the project's platform be better? For example:

```
        for project in Project.objects.filter(organization=org, status=0):
            try:
                if project.platform == "node-tracing":
                    project.platform = "node"
                    project.save()
                    continue

                if project.platform == "python-tracing":
                    project.platform = "python"
                    project.save()
                    continue
            except Exception:
                logging.exception("Error changing platform for project {}".format(project.id))
```

Reply:
Yeah, we typically do that for this kind of case. It's maybe not such a huge deal if you're already filtering on the org though, since there aren't that many projects per org.